pulsar-client/pom.xml

```diff
@@ -136,6 +136,12 @@
       <version>${project.parent.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>${joda.version}</version>
```

Comment:
Once the dependency is set in `dependencyManagement`, we won't need to specify the version here.